cms/djangoapps/contentstore/management/commands/tests/test_export_all_courses.py

```diff
@@ -20,8 +20,12 @@ def setUp(self):
         """ Common setup. """
         self.store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
         self.temp_dir = mkdtemp()
-        self.first_course = CourseFactory.create(org="test", course="course1", display_name="run1")
-        self.second_course = CourseFactory.create(org="test", course="course2", display_name="run2")
+        self.first_course = CourseFactory.create(
```

Comment:
You could use the `with modulestore().default_store(...)` context manager to wrap the whole block.